spec/features/tour_list/create_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'tour list create', :vcr, type: :feature do
```

Comment:
Confusing file names -- should this be something about a recording instead?

Reply:
I don't think it should be about a recording. The feature is the ability to create a tour. The user is able to generate a list of landmarks for their tour. Adding landmarks and adding recordings to those landmarks are parts to the functionality of the tour list feature. Therefore, the file name is tour list: create but VCR does not like symbol so it may look confusing.